opds.py

```diff
@@ -582,6 +602,21 @@ def page(cls, _db, title, url, lane, annotator,
             pagination.this_page_size = len(works)
         feed = cls(_db, title, url, works, annotator)
 
+        if lane.entrypoints and (not pagination or pagination.offset == 0):
+            # On the first page of a paginated feed there may be
+            # multiple entry points into the same dataset.
```

Comment:
This could lead to a weird UI where tabs disappear when you go to the page 2, depending on how the client works. If all the clients do infinite scroll instead of pagination it would be ok.

Reply:
I left that link out because I'm not sure where a tab on page 2 would go. Would it go to the first or second page of the other tab? Neither seems great to me.
 
I guess if it's like other facet groups, it would go to the top of the other tab. I can take out the `pagination.offset == 0` stuff if you'd like that behavior.

Reply:
I do think that would be better.